src/wasm-binary.h

```diff
@@ -116,6 +116,7 @@ struct LEB {
   }
 };
 
+#define SIZE32LEB(v) U32LEB((uint32_t)(v))
```

Comment:
I agree on not doing more in this PR. Perhaps let's take out the SIZE32LEB then, and later think more on the types in this file?